spec/controllers/comments_controller_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'rails_helper'
+
+RSpec.describe CommentsController, type: :controller do
+  it { is_expected.to use_before_action(:authenticate_user!) }
+
+  let!(:post_) { FactoryGirl.create(:post) }
+
+  describe 'GET #new' do
+    login_user
+    before { get :new, post_id: post_ }
+
+    it { is_expected.to render_template 'new'}
+
+    it 'assigns the post to @post' do
+      expect(assigns :post).to eq post_
+    end
+
+    it 'assigns a new Comment to @comment' do
+      expect(assigns :comment).to be_a_new(Comment)
```

Comment:
Style/NestedParenthesizedCalls: Add parentheses to nested method call `assigns :comment`.